scripts/zones/Upper_Jeuno/npcs/Mapitoto.lua

```diff
@@ -30,4 +67,16 @@ function onEventUpdate(player,csid,option)
 end
 
 function onEventFinish(player,csid,option)
+    if csid == 10223 and option == 1 then
+        player:setPos(475, 8.8, -159, 128, 105)
+    elseif csid == 10225 then
+        -- Complete quest
+        npcUtil.giveKeyItem(player, tpz.ki.TRAINERS_WHISTLE)
+        npcUtil.giveKeyItem(player, tpz.ki.RAPTOR_COMPANION)
+    elseif csid == 10227 then
+        player:confirmTrade()
+        local rewardKI = player:getCharVar("FullSpeedAheadReward")
+        player:setCharVar("FullSpeedAheadReward", 0)
```

Comment:
```lua
local rewardKI = player:getLocalVar("FullSpeedAheadReward")
player:setLocalVar("FullSpeedAheadReward", 0)
```

Reply:
I need to go read the difference between LocalVar and CharVar, thanks!